scripts/release/install.js

```diff
@@ -62,6 +63,23 @@ module.exports = {
         // up exception may contain a passphrase.
         console.log("Error building win32 installer")
       })
+  },
+
+  debian: function() {
+    console.log("Building installer for debian")
```

Comment:
You resolved my comment but did not make the suggested change?

Reply:
I committed your suggestion but it got wiped out by my force push.

Reply:
oops! :)